### PR TITLE
Sx8 view notes

### PR DESCRIPTION
**Pull request by ahmad-diaa**
#55

Reviewers  :-
@Youmna-Salah:  code
@daliamaarek: scenario
@maggie-moheb: test
@azamzamy: documentation
@AmirMaurice: UML

### CODE REVIEW

**Comment by ahmad-diaa** [comment]
@Youmna-Salah done!

**Comment by daliamaarek** [comment]
if you write a huge note, the text field overlaps the "add note" button.

**Comment by ahmad-diaa** [comment]
@daliamaarek  this problem is in AddNote, my story is viewNotes.
@maggie-moheb  add note issue should be reopened

**Comment by AmirMaurice** [comment]
@ahmad-diaa your sequence diagram and wireframe are great but I couldn't find your class diagram

File: SmartX/app/src/main/java/com/smartx/cookies/smartx/test/viewNotesActivityTest.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+     */
+    public void testViewNotes() throws Exception {
+        myActivity.runOnUiThread(new Runnable() {
+            public void run() {
+                Notes.add("Please don't close it");
+                Notes.add("Keep the volume down");
+            }
+        });
+
+
```
Comment (azamzamy) [inline SmartX/app/src/main/java/com/smartx/cookies/smartx/test/viewNotesActivityTest.java:79]: @Youmna-Salah Empty line. Please take care next time!!!!

**Comment by azamzamy** [comment]
@ahmad-diaa Documentation is good, well done.

**Comment by ahmad-diaa** [comment]
@AmirMaurice I added the class diagram.

**Comment by AmirMaurice** [comment]
@ahmad-diaa :+1:

**Pull request merged**